calculator.go

```diff
@@ -1,13 +1,125 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes a list of numbers and returns the result of adding them together.
+func Add(nums ...float64) float64 {
```

Comment:
This is great, but one potential problem is that we could pass zero or one arguments to Add, which doesn't really makes sense! Could we write the parameter list in such a way that the compiler will require _at least two_ arguments, and maybe more?